packages/plugin-ext/src/main/node/plugin-localization-backend-contribution.ts

```diff
@@ -0,0 +1,35 @@
+import * as express from 'express';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { LocalizationBackendContribution } from '@theia/core/lib/node/i18n/localization-backend-contribution';
+import { Deferred } from '@theia/core/lib/common/promise-util';
+import { PluginDeployer } from '../../common/plugin-protocol';
+import { PluginDeployerImpl } from './plugin-deployer-impl';
+
+@injectable()
+export class PluginLocalizationBackendContribution extends LocalizationBackendContribution {
+  @inject(PluginDeployer)
```

Comment:
Minor: We use a tab width of 2 spaces across Theia. Please align this file to this guideline.